cluster/aws-redshift-cluster.json

```diff
@@ -153,15 +155,16 @@
         "IamRoles": {
             "description": "A list of AWS Identity and Access Management (IAM) roles that can be used by the cluster to access other AWS services. You must supply the IAM roles in their Amazon Resource Name (ARN) format. You can supply up to 10 IAM roles in a single request",
             "type": "array",
+            "insertionOrder": false,
             "maxItems": 10,
-            "uniqueItems": true,
```

Comment:
we don't need the uniqueness check here any more?

Reply:
If we keep `uniqueItems` attribute, the ResourceModel will define `IamRoles` as `Set` instead of `List`, which is incompatible to the current Handlers' usage. Furthermore, the unique entity checking can be resolved by the API Front-End, meaning this validation is not necessary.